packages/client/src/client4.ts

```diff
@@ -548,7 +548,7 @@ export default class Client4 {
 
         if (options.body) {
             // when the body is an instance of FormData we let browser set the Content-Type header generated by FormData interface with correct boundary
-            if (!(options.body instanceof FormData)) {
+            if (!options.body) {
```

Comment:
This does not work with Playwright being node client when accessing this API. I wonder if this could be removed or if `form-data` could be reverted to support this condition.

Reply:
so how will reverting form-data help here?

Reply:
First, in latest commit, I reverted the change to `if (!(options.body instanceof FormData)) {...}` and add `form-data` package again. Let me know if that's fine or if there's any workaround.

When client package is used in node client, no `FormData` is available since it's on browser only. Having `form-data` as dependency helps fill the gap and make the client package usable not only by the webapp but likely by other application that would like to integrate with Mattermost.

Reply:
Oh it makes sense, i am ok with adding it back. This was the pr which removed it https://github.com/mattermost/mattermost-webapp/pull/11723